src/sql/src/plan/statement/ddl.rs

```diff
@@ -3362,7 +3362,7 @@ fn ensure_cluster_is_not_linked(
 }
 
 fn is_storage_cluster(scx: &StatementContext, cluster: &dyn CatalogCluster) -> bool {
-    cluster.bound_objects().iter().all(|id| {
+    cluster.bound_objects().iter().any(|id| {
```

Comment:
Also wrote this somewhere but not sure if it will show up: the commit message mixes up `any` and `all` :sweat_smile:

Reply:
Oh, I remember now why I used `all`. `all` is false on empty iterators, whereas `any` is true on empty iterators. And we want the former behavior.

Reply:
https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.all

`all` is `true` on empty iterators!

Reply:
Gah, sorry, fixed.